src/main/kotlin/io/github/mojira/arisa/modules/CrashModule.kt

```diff
@@ -12,6 +12,7 @@ import io.github.mojira.arisa.infrastructure.AttachmentUtils
 import io.github.mojira.arisa.infrastructure.config.CrashDupeConfig
 import me.urielsalis.mccrashlib.Crash
 import me.urielsalis.mccrashlib.CrashReader
+import java.io.File
 import java.time.Instant
 
 class CrashModule(
```

Comment:
It might make sense to split this module up at some point, it does so many things at once. (But out of scope for this PR)